### DISCUSSION TITLE
Create `retry latest version` label on this repository

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#10240] [Suggest an Idea]
I made a PR which makes it so that the label action bot will comment on `retry latest version` label:

https://github.com/renovatebot/renovate/blob/675b8aac9e706df82608a6b8c24913cae9bdc8cf/.github/label-actions.yml#L122-L134

The label does not exist in the repository label list.

Can you please create the label? 😉

**Comment by rarkins** [comment]
Done

**Reply by HonkingGoose** [reply]
Thank you! 😄
